src/Quartz/Type/No.elm

```diff
@@ -0,0 +1,15 @@
+module Quartz.Type.No exposing (No, no)
+
+
+type No
+    = No No
+
+
+no : No -> a
+no (No x) =
+    let
+        y : No
+        y =
+            x
+    in
+    no y
```

Comment:
I defined this in such a weird way to prevent elm-review from complaining about it.